cereal/car.capnp

```diff
@@ -50,6 +50,7 @@ struct CarState {
   struct CruiseState {
     enabled @0: Bool;
     speed @1: Float32;
+    mainOn @2: Bool;
```

Comment:
@geohot , I've been thinking about where to put this indeed. It belongs either here or in the parent structure CarState. When Main is off, pcm cruise can't be engaged. Main is a latching state that is sent on CAN by the steering control module; it can be altered by pressing the "Main" button on the steering wheel.